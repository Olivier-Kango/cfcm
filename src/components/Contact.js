import React from 'react';

const Contact = () => (
  <div className="contact">
    Contact
  </div>
);

export default Contact;
