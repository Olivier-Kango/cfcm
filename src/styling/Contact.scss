.contact {
  padding: 6rem 5%;
  text-align: center;
  height: 100vh;
  color: white;
  background-image: url("../images/black-brick-bd.jfif");

  span {
    text-decoration: underline;
  }

  .social-icons {
    a {
      font-size: 24px;
      margin-right: 15px;
      color: #f5f5f5;
      text-decoration: none;
      transition: color 0.3s;
      display: inline-block;
      width: 40px;
      height: 40px;
      line-height: 40px;
      text-align: center;
      border-radius: 50%;
      background-color: rgba(255, 255, 255, 0.1);

      &:not(:last-child) {
        margin-right: 15px;
      }

      &:hover {
        background-color: rgba(255, 255, 255, 0.2);
        color: #fff;
      }
    }

    .facebook:hover {
      color: #1877f2;
    }

    .youtube:hover {
      color: #f00;
    }
  }

  .social-media {
    margin-top: 30px;
    flex: 1;
    text-align: center;
  }

  h4 {
    margin-bottom: 10px;
    color: #f5f5f5;
  }

  .copyright p {
    position: absolute;
    bottom: 0;
    left: 0;
    right: 0;
    text-align: center;
    color: white !important;
    padding: 0;
    margin: 0 !important;
    margin-top: 20px;
    padding-bottom: 20px;
  }
}
