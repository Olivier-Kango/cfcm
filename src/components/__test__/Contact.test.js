import { render, screen } from '@testing-library/react';
import Contact from '../Contact';

describe('Testing the Contact page', () => {
  test('should render', () => {
    render(<Contact />);
    const about = screen.getByText(/Contact/i);
    expect(about).toMatchSnapshot();
  });
});
