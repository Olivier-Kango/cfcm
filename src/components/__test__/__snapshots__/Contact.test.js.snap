// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Contact page should render 1`] = `
<div>
  Contact
</div>
`;
